includes/settings/js/src/components/Tools.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { __ } from "@wordpress/i18n";
+import ExportFileButton from "./ExportFileButton";
+const { wp } = window;
+const { apiFetch } = wp;
+
+export default function Tools() {
+	/**
+	 * Gets model export data via the REST API.
+	 */
+	function getModels() {
+		return apiFetch({
+			path: `/wpe/atlas/content-models/`,
+			method: "GET",
+			_wpnonce: wpApiSettings.nonce,
+		});
+	}
+
+	/**
+	 * Format filename for export
+	 * @returns {string}
+	 */
+	function getFormattedDateTime() {
+		var today = new Date();
+		var y = today.getFullYear();
+		var m = today.getMonth() + 1;
+		var d = today.getDate();
+		var h = today.getHours();
+		var mi = today.getMinutes();
+		var s = today.getSeconds();
+		return m + "-" + d + "-" + y + "-" + h + "-" + mi + "-" + s;
+	}
```

Comment:
Worth considering a standard format like [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) with year-month-day?

For example:

```suggestion
	function getFormattedDateTime() {
		return new Date().toISOString().split('.')[0].replace(/[T:]/g, '-');
	}
```

Which gives:

```
2021-09-20-09-49-38
```

It's generally easier to sort/parse filenames in this format because sorting by name also sorts by date.